test/convertors.js

```diff
@@ -103,5 +103,10 @@ describe('Convert data from one type to another', function () {
           .to.throw(TypeError, 'Wrong data type passed to convertor. Hexadecimal string is expected')
       })
     })
+
+    it('should convert hexadecimal to BinaryString and back', function () {
```

Comment:
Nitpick: why is `BinaryString` written in camel case?

Reply:
@slowli really weird